src/modules/RA/DataSource/components/ReportTab.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import { FixedSizeList } from 'react-window';
-import { List, ListItem, ListItemText, ListItemIcon, makeStyles } from '@material-ui/core';
-import Divider from '@material-ui/core/Divider';
+import { List, ListItem, ListItemText, ListItemIcon, makeStyles, Divider, Typography } from '@material-ui/core';
```

Comment:
Doesn't linting rules enforce wrapping for lines longer than 100 chars?

```suggestion
import {
  Divider,
  List,
  ListItem,
  ListItemIcon,
  ListItemText,
  makeStyles,
  Typography,
} from '@material-ui/core';
```